tests/integration/kubernetes/run_kubernetes_tests.sh

```diff
@@ -62,6 +62,10 @@ if [ ${KATA_HYPERVISOR} == "qemu-snp" ]; then
 	exit 0
 fi
 
+if [ ${KATA_HOST_OS} == "cbl-mariner" ]; then
```

Comment:
Is there a way to probe k8s to determine the node's OS so that we don't need to introduce another environment variable?

Reply:
We could do that by extracting the `osImage` field from the node description indeed, however the value returned by that field on Mariner isn't stable yet, so to avoid inadvertently breaking the CI with a new Mariner release, I think it's better to keep this environment variable for now.

Reply:
checking `/etc/os-release` contents would also work

Reply:
Yes, I believe that's how `osImage` is sourced. However, with the new [Azure Linux](https://aka.ms/azurelinux), it's still TBD what that file would contain longer term.